cypress/e2e/tests/administration/questionnaires/crud.test.ts

```diff
@@ -26,6 +32,11 @@ describe(["@tier2"], "Questionnaire CRUD operations", () => {
         closeModalWindow();
     });
 
+    it("Export questionnaire", function () {
+        AssessmentQuestionnaire.export(fileName);
+        cy.readFile("cypress/downloads/questionnaire-4.yaml").should("contain", "Cloud Native");
```

Comment:
Let's use fileName paramter here so that if anything changes we will just change the paramter globally .

Reply:
Done